meerk40t/device/lhystudios/lhystudiosemulator.py

```diff
@@ -327,4 +327,5 @@ def load(kernel, pathname, **kwargs):
                 "module/LhystudiossEmulator", basename
             )
             lhymicroemulator.write_header(f.read())
-            return [lhymicroemulator.cutcode], None, None, pathname, basename
+            kernel.root.close(basename)
+        return True
```

Comment:
I'm not sure this already added itself to the data tree.

Reply:
True means successful - a file_node means added to the tree.